time/chrono.h

```diff
@@ -65,8 +65,8 @@ inline EPOCH_MILLISECONDS Now() {
 }
 
 #else
+#warning 'thread_local' is not supported by compiler
```

Comment:
Would it be better to put this `#warning` into `port.h`? This way, the user would get it once per build.